vividus-facade-jira/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
where is it used?

Reply:
to mock JiraConfigurationProvider in JiraClientProvider